app/assets/stylesheets/dashboard.scss

```diff
@@ -7,6 +7,10 @@
     background: $bold-blue;
   }
 
+  .organizations-menu {
+    list-style-type: none;
+  }
+
```

Comment:
suggestion (not blocking) - instead of creating a new class, you could use [our utility class for list-style-none](https://storybook.forem.com/?path=/story/utility-first-classes-list-style-type--list-none)